t/decode-transactions

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# For every create-withdrawal-data.*.golden, use bitcoin-cli
+# "decoderawtransaction" to dump the contents of the constructed
+# transaction.
+#
+# I'm using this to document the changes that Bitcoin Core v0.17 is
+# making to the transactions (shorter signatures thanks to
+# https://github.com/bitcoin/bitcoin/pull/13666)
+#
+
+bitcoind -testnet -daemon -connect=0.0.0.0
+# Give bitcoind time to start up
+sleep 5
+
+
+
+for i in create-withdrawal-data.*.golden; do
+    # Find line following "Raw signed transaction (hex):"
+    MATCH=false
+    HEX=
+    while read -r line; do
+        if [ "$MATCH" = true ]; then HEX=$line; MATCH=false; fi
```

Comment:
this should have a `break` - or we should collapse these two checks into one line if possible, though I don't have a particular bash solution in mind